modules/phase_field/include/kernels/SplitPFFractureBulkRate.h

```diff
@@ -0,0 +1,38 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#ifndef SPLITPFFRACTUREBULKRATE_H
+#define SPLITPFFRACTUREBULKRATE_H
+
+#include "PFFractureBulkRateBase.h"
+
+// Forward Declarations
+class SplitPFFractureBulkRate;
+
+template <>
+InputParameters validParams<SplitPFFractureBulkRate>();
+
+/**
+ * Phase field based fracture model, split form where beta = laplace(c)
+ * This kernel computes the residual and Jacobian for bulk free energy contribution to c
+ * Refer to Formulation: Miehe et. al., Int. J. Num. Methods Engg., 2010, 83. 1273-1311 Equation 63
+ */
+class SplitPFFractureBulkRate : public PFFractureBulkRateBase
+{
+public:
+  SplitPFFractureBulkRate(const InputParameters & parameters);
+
+protected:
+  virtual Real precomputeQpResidual();
```

Comment:
Could the "PFFractureBulkRateBase" class derive from Kernel instead of KernelValue? This will give user more flexibility to implement other PFFractureBulkRate kernel which might include grad_test term.